tests/MSBuildDeviceIntegration/Tests/InstallTests.cs

```diff
@@ -217,7 +217,11 @@ public void InstallWithoutSharedRuntime ()
 				//	"The Shared Runtime should not have been installed.");
 				var directorylist = GetContentFromAllOverrideDirectories (proj.PackageName);
 				StringAssert.Contains ($"{proj.ProjectName}.dll", directorylist, $"{proj.ProjectName}.dll should exist in the .__override__ directory.");
-				StringAssert.Contains ($"System.dll", directorylist, $"System.dll should exist in the .__override__ directory.");
+				if (Builder.UseDotNet)
+					StringAssert.Contains ($"System.Private.CoreLib.dll", directorylist, $"System.Private.CoreLib.dll should exist in the .__override__ directory.");
```

Comment:
If this test fails, you might check for a different assembly like `System.Linq.dll`. I think `System.Private.CoreLib.dll` can be in a sub-directory like `armeabi-v7a` or `x86`.

Reply:
System.Private.CoreLib.dll is probably only reliable marker. I hope `System.Linq.dll` will be gone too next week ;-)